src/DI/Helpers.php

```diff
@@ -98,7 +99,7 @@ public static function autowireArguments(\ReflectionFunctionAbstract $method, ar
 				unset($arguments[$parameter->getName()]);
 				$optCount = 0;
 
-			} elseif ($class = PhpReflection::getParameterType($parameter)) { // has object type hint
+			} elseif ($class !== NULL && !in_array(strtolower($class), ['string', 'int', 'float', 'bool', 'array', 'callable'], TRUE)) { // has object type hint
```

Comment:
Yeah, I was thinking the same (probably `isBuiltinType` to be consistent with PHP 7 naming), since the same `in_array` is used few lines lower in `expandClassName`. But I'm pretty new to Nette PRs and I'm not sure if such refactoring is acceptable in PR with a bugfix?